irohad/main/impl/pg_connection_init.cpp

```diff
@@ -279,15 +279,28 @@ CREATE TABLE IF NOT EXISTS account_has_grantable_permissions (
       + R"() NOT NULL,
     PRIMARY KEY (permittee_account_id, account_id)
 );
-CREATE TABLE IF NOT EXISTS position_by_hash (
-    hash varchar unique not null,
+CREATE TABLE IF NOT EXISTS tx_positions (
+    creator_id text,
+    hash varchar(64) not null,
```

Comment:
what if we use sha512?
```suggestion
    hash varchar(128) not null,
```